ops/ccs-ops-misc/load_test/v2/claimresponse_test_hashedMbi.py

```diff
@@ -0,0 +1,32 @@
+import urllib3
+import common.config as config
+import common.data as data
+import common.errors as errors
+import common.test_setup as setup
+from locust import HttpUser, task
+
+server_public_key = setup.loadServerPublicKey()
+
+'''
+If there is no server cert, the warnings are disabled because thousands will appear in the logs and make it difficult
+to see anything else.
+'''
+if not server_public_key:
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+mbis = data.load_pa_mbis()
+client_cert = setup.getClientCert()
+setup.set_locust_env(config.load())
```

Comment:
Can any of this boilerplate be consolidated into a module?

Reply:
Possibly, I'm really not well versed with python though, so I was just following the convention of the other tests

Reply:
what do you think about consolidating all of this into 2 files, `claim.py` and `claimresponse.py`.  In each file we could have different tasks for running each query variation and [tag the tasks](https://docs.locust.io/en/stable/writing-a-locustfile.html#tag-decorator) with different labels to give us the ability to isolate them?